pom.xml

```diff
@@ -147,14 +147,24 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-checkstyle-plugin</artifactId>
-          <version>3.1.2</version>
+          <version>2.17</version>
           <configuration>
-            <configLocation>google_checks.xml</configLocation>
+            <configLocation>checkstyle.xml</configLocation>
             <encoding>UTF-8</encoding>
             <consoleOutput>true</consoleOutput>
             <failsOnError>true</failsOnError>
+            <failOnViolation>true</failOnViolation>
+            <includeTestSourceDirectory>true</includeTestSourceDirectory>
             <linkXRef>false</linkXRef>
+            <excludes>com/sun/**/*,org/apache/maven/**/*,edu/illinois/starts/asm/**/*</excludes>
```

Comment:
remove this line 159; we no longer need it.

In fact, if you can make a PR to remove it from STARTS's master branch, that would be appreciated

Reply:
Yes, please remove `com/sun/**/*,org/apache/maven/**/*`